### DISCUSSION TITLE
Custom data (custom properties) attached to pageviews

### DISCUSSION THREAD

**Discussion by davidfischer** [plausible/analytics#320] [Anything Else]
## Precheck

Please note that this tracker is only for feature requests. Do not use the issue tracker for help or support. [Our docs](https://docs.plausible.io/) are a great place for most answers, but if you can’t find your answer there, you can [contact us](https://plausible.io/contact). Have an issue with your self-hosted install? You can ask in [our community forum](https://plausible.discourse.group/). Thanks!

## Prerequisites
- [X] I have searched open and closed issues to make sure that the feature has not yet been requested.

## Feature request
**Is your feature request related to a problem? Please describe.**
I would like to be able to attach custom data (think [GA custom dimensions](https://support.google.com/analytics/answer/2709828?hl=en)) when sending a pageview. Then I would like to aggregate on those dimensions. I saw the documentation on [custom events](https://docs.plausible.io/custom-event-goals) but I didn't see anything on attaching custom data to a pageview.

**How would you like it to work**
I'm pretty flexible here. Perhaps the custom data could come from custom meta tags or perhaps there would be an option to manually trigger a pageview with JavaScript and send the data that way.

Sorry if this feature is available somewhere else and I couldn't find it.

**Update:** We've now opened up a beta version of this to everyone. You can attach custom properties (also known as custom dimensions in Google Analytics) when sending a pageview to Plausible in order to create custom metrics. Instructions at https://plausible.io/docs/custom-pageview-props

**Comment by ukutaht** [comment]
Agreed, this would make the platform a lot more powerful.

One use-case we've been asked before is filtering articles by the author of the post. Instead of implementing specific fields like `author`, it would probably make more sense to allow flexible custom dimensions. See #302 

I'm really open for suggestions on how the API for this could look. Some options off the top of my head:

#### 1. Manual

In this case, to send a custom dimension you need to disable automatic pageview collection and instead send them manually along with your custom dimensions:

```html
<script async defer data-manual="true" data-domain="plausible.io" src="https://plausible.io/js/plausible.js"></script>
```

... later

```javascript
plausible('pageview', custom: {author: 'Uku Taht'})
```

#### 2. Automatic via script tag

This option would probably be less flexible but more convenient depending how you render your page. Just set the custom dimension via html attributes on the script tag:

```html
<script async defer custom-author="Uku Taht" data-domain="plausible.io" src="https://plausible.io/js/plausible.js"></script>
```

The tracker could check for any attribute following the `custom-{key}="{value}"` pattern and automatically send them along.

**Reply by gravelld** [reply]
This would be very useful.

For consistency, why not use the same JSON member name as custom properties in goals (so `props` rather than `custom`), i.e.

    plausible('pageview', {props: {author: 'Uku Taht'}})
    
As per https://plausible.io/docs/custom-event-goals#using-custom-props

**Reply by ukutaht** [reply]
Totally. I think I wrote this comment before we implemented custom props for custom events. Now we need to extend it to pageviews as well.

**Comment by davidfischer** [comment]
Another option as I mentioned is to have custom meta tags that are sent. This could work in a somewhat similar fashion to something like opengraph. Think tags like


```html
<meta property="analytics:author" content="David" />
<meta property="analytics:another_field" content="Another Value" />
```

Any tags with the appropriate prefix get picked up and sent as custom dimensions. These same pieces of data could get picked up for custom events as well.

Something like your manual proposal could work as well.

**Comment by ukutaht** [comment]
Right, that looks like a great option as well.

**Comment by janejuenyang** [comment]
Hello! Glad to see this feature request has already been raised. :) Chiming in to mention another use case for this feature: A/B testing. To make sure we have a large enough sample size for a test, we’d need to know how many visitors saw a particular variation.

FWIW on the API form, we’re also flexible but would prefer the data-attribute or meta tag forms over the manual configuration.

**Comment by ukutaht** [comment]
@janejuenyang Yeah thanks for bringing that up, we don't have a good way to segment visitors for A/B testing.

I think implementing custom props on pageviews will definitely be helpful. You can tag the pageview with the visitor's segment and also tag the conversion event. It will work for getting conversion rates but there's some manual work involved where you have to grab the numbers and plug them into a calculator.

A second phase could be full support for A/B testing. This would involve:
1. Adding a session-level `segment` property to capture which variant the user is shown
2. Making Plausible aware of A/B testing so we can calculate the conversion rates and confidence intervals for you

**Comment by morizyun** [comment]
Looks so great!!

This feature would help our marketing tremendously! I'd love to see you put this feature on your roadmap!

**Reply by ukutaht** [reply]
Thanks for letting us know!

May I ask what your specific use-case is so I can take that into account when building this feature?

**Comment by Adamkillander96** [comment]
The first suggestion @ukutaht provided looks great!

As for use cases, custom dimensions is great for many reasons. 

Examples:
- You have a website where you can sign in. Having a dimension simply called `login_status` (a Boolean) to filter on signed in or out users. 
- A `user_role` dimension, like guest, admin, subscriber, etc... 
- Country and language dimensions, if your site/app does not have the country or language in the URL, dimensions helps with filtering on that.
- I personally like to track PWA usage. Having a dimension like `visit_platform` and populate it with 'app' for PWA visits and 'browser' for normal usage.

As previously mentioned, the author name example, page information is very important. Like:
- Page type: home, product, article, faq, etc...
- Page category: if you run a blog or something similar and want to see which category performs best
- Page id: something like page name can change and is not good practice if you have a multi language site. So identifying a post, page or product by ID helps filtering views and events across different URLs and languages. 

Say you have 3 languages and a specific product you want to look at. If the url structure is something like this:

`https://store.com/{language}/product/{product_name}`

You are going to have to do some extra work to see how many visits product A has, because it has three different URLs.
With a page ID, we could see how many pageviews it has, regardless of language, country or it's translated name. 

These are some of the offhand use cases I can think of with custom dimensions.

**Reply by ukutaht** [reply]
Hey @Adamkillander96 ! We're currently working on the custom pageview prop feature (will be named custom dimensions when we're ready). Just wanted to get your ideas on what the best way to send these props would be? Do you use Google Tag Manager or DataLayer?

**Reply by brandonburkett** [reply]
While not Adam, I would use it like this in UA: `ga('set', 'dimension1', 'Free Member')`, then all pageviews / events, etc get the custom dimension association without having to manually set it.

**Comment by anderstp** [comment]
I think this option would help a problem I am facing with our usage of plausible.

We are building an app where users signs up they choose a username and then using that as part of the url, ex. would be maelt.co/anda (my co-founder)

The thing is, we want to offer (already do in beta) to point your domain to our server, and just show the profile page as the root of the domain - in this case it would be anda.co (not working) and in plausible it then shows as "/"

By being able to add a custom user-id tag to the plausible tag, we would be able to see the difference between visitors to our homepage and to users pages on custom domains.

**Reply by ukutaht** [reply]
@anderstp We already collect hostname information on each pageview but we do not surface it in the UI. This could be used to discern between a user on maelt.co/anda vs anda.co/.

More info and discussion: #160

**Reply by anderstp** [reply]
I agree, that would solve my headache :)

**Comment by justrealmilk** [comment]
For PWA apps, I'd love a way to track users' upgrade journey. Sending the app version along could help me monitor when users are stuck on an old version for example.

**Comment by metmarkosaric** [comment]
"we set up the following Google Analytics Custom Dimensions on our story pages, via variables provided by our CMS. This allows us to segment traffic based on the content itself. We'd like to continue to segment our content as we move away from Google Analytics. 

Dimensions:
1. Author Byline
2. Program Taxonomy
3. Publish date
4. Tags
5. Categories 
6. The internal classification for the story
7. The story template style"

(one more vote for this feature today via email)

**Comment by OriginalEXE** [comment]
+1 from me.

I have recently implemented dark mode on my website and would love to track how many people use it. Enriching automatic page views 
tracking with something like `darkMode: true/false` would be great.

I could of course track toggling of the dark mode, but that doesn't really give me an accurate representation of how many people use light vs dark mode.

Another info I would like to track is whether my visitors are logged in or not, so again a boolean/string attached to all page views.

**Reply by FiloSottile** [reply]
Same here, I wanted to track how many users have dark mode enabled system-wide to decide if it's worth supporting.

**Comment by janegbert** [comment]
I would also +1 this  idea. It is something we do in Google Analytics with content groups and custom dimesions. I would like to be able to label pageviews by:
- page type (index, singular, etc.)
- field of knowledge
- industry vertical
- content type
- author
- year of publication

The big advantage of this is that it allows us to easily report on developments in analytics for these custom properties. Making it much easier to make data-driven editorial decsision.

**Reply by ukutaht** [reply]
Thanks @janegbert! We are finalizing this feature so soon you should be able to send these dimensions. What would be the most convenient way to send it for you? Do you use GTM or dataLayer?

**Reply by janegbert** [reply]
@ukutaht Thanks for working on this feature. In our case I would like to send the dimensions straight from the page HTML / tracking script because we would like the implementation be independent from Google services.

**Comment by itsmatteomanf** [comment]
I am going to add my support for this feature. My use case is the same as some of the above.

- Multi-language websites, with same products.
- Seeing per language traffic or other stat.

**Comment by metmarkosaric** [comment]
WordPress related but feels like it fits for this feature request: "We’re using WPML to translate our site. Somehow Plausible should be able to separate between languages used on the website. Could be done over hreflang or similar. So we can filter the data based on languages and additionally see which languages get used how often."

**Comment by aresundnes** [comment]
+1 from me on this! Would help us out with AB testing

**Comment by gravelld** [comment]
Is it possible to use https://plausible.io/docs/events-api as a workaround, using the `props` member?

Something like:

    curl -i -X POST https://plausible.io/api/event \
      -H 'User-Agent: ... \
      -H 'X-Forwarded-For: 127.0.0.1' \
      -H 'Content-Type: application/json' \
      --data '{"name":"pageview","url":"http://dummy.site","props":"%7Bmethod%3A%20%27HTTP%27%2C%20lang%3A%20%27en%27%7D"}'

**Comment by reustle** [comment]
+1 for this feature

We'd love to track the visitor's browser language via `window.navigator.language`, to see how the traffic splits across multiple native languages (i.e. `en_US`).

We already have default plausible filter options to see some similar info from the browser, but it's missing `language` so a custom field would be required. Using a `goal` in this case won't really help us it seems. Thanks so much.

![image](https://user-images.githubusercontent.com/304560/151685940-0c7d731e-92c5-47e0-849a-83cefd6fb599.png)

**Comment by metmarkosaric** [comment]
Another request for this via email with this use case:

"I don't see a way in Plausible Analytics to group pageviews, as done with content groups in Google Analytics. On our web site, we want to count pageviews for pages like site.com/name2 and site.com/something-else together, because these are of the same page type. But we also want to count each of these urls separately.

In GA it is done like this:
ga('set','contentGroup1','name of content group')

Basically I just want to set a parameter that describes what type of page it is.

This way we can set a name for each type of page. We have millions of pages, and it makes sense to count them as one type of page."

**Reply by mquandalle** [reply]
We have the same use case

**Comment by rmens** [comment]
+1 for this. I'm responsible for some dashboards for a large publisher and use lots of custom dimensions in GA. The option to also have them in Plausible would make it a more 'enterprise ready' GA alternative.

This are some of the dimensions I work with daily.

- Author
- Pubdate
- Categories
- Tags
- Words
- Type
- PostID
- IsAdvertorial

**Reply by ukutaht** [reply]
Hey @rmens thanks for your input! Could you also share how these publishers implement these dimensions? Do they use GTM or dataLayer? What would be the most convenient way for you to send these to Plausible?

**Reply by rmens** [reply]
@ukutaht Currently it's mostly implemented as a dataLayer.

**Reply by ukutaht** [reply]
@rmens We're finalizing some of the early work we're doing on this feature. I have a question – can you send a list of values as a dimension in Google Analytics? I would imagine that things like 'categories' and 'tags' are lists, but as far as I can tell you cannot send lists to GA: https://stackoverflow.com/questions/36360165/how-to-send-an-array-as-google-analytics-custom-dimension

**Reply by ukutaht** [reply]
I suppose one can send a comma-separated string instead and use the `contains` filter as proposed [here](https://github.com/plausible/analytics/discussions/320#discussioncomment-5936342)?

**Comment by repolevedavaj** [comment]
We want to do something similar: We do not want to track the users itself but the organization they work for. It would be helpful if we could configure a callback to attach additional `props` to the `pageview` event. This would allow us to configure the `pageview` event as goal and then check the calls by organization. This seems only to be possible by patching the js.

**Comment by davidmondok** [comment]
We are very much looking forward to such a feature in Plausible. The dimensions that we are working with are basically the same as @rmens listed in his last comment.

**Comment by tf** [comment]
This issue is marked as "in progress" on the roadmap. Some related commits appear to already be on master. Is there already an ETA for this feature?

**Reply by metmarkosaric** [reply]
hi @tf! that's right. the development is in progress and we're already testing things. sorry but i don't have an ETA at the moment. as soon as it's ready, we'll post in this thread. thanks for your patience!

**Comment by daffyquack** [comment]
This feature would allow to record personal data in your database, right? Nothing prevents from storing email adresses, user ids etc...
Wouldn't this damage the privacy-friendlyness of the service?

**Reply by metmarkosaric** [reply]
thanks for the feedback @daffyquack! the site owner would be responsible to ensure that it doesn't happen. this feature would work the same way as custom dimensions that people can already attach to custom events. you can read more here: https://plausible.io/docs/custom-event-goals#using-custom-props

**Comment by jlivak** [comment]
We're currently migrating our site from GA over to Plausible and custom dimensions for pageviews would be great to have.  Our use case is the same as people mentioned above, distinguishing between free/paid/unregistered users.

**Comment by brandonburkett** [comment]
With GA4-mageddon incoming, this is one (of three) features we are looking for as we review privacy focused options.  The others are funnels and events.

Our dimensions use case is pretty simple, segmenting page views / metrics by anon, free, subscriber user roles.

Looking forward to an ETA (looks like it has been a roadmap feature for a long time) so I can help make the case for plausible!

**Reply by irtimood** [reply]
Same here, this feature is necessary too let us migrate to plausible

**Comment by ivmilicevic** [comment]
I see that pull request #1816 (https://github.com/plausible/analytics/pull/1816) was merged and referenced in changelog for v1.5.0.

Is this feature now usable or is there more work to do?

**Reply by metmarkosaric** [reply]
hi @ivmilicevic! this is not yet ready. it is the next big feature for us to complete and announce. stay tuned!

**Reply by mattcameron** [reply]
Do you have a rough idea of when this feature might be released? Plausible won't be suitable for our needs without it and we need to move away from GA asap.

**Reply by metmarkosaric** [reply]
hi @mattcameron! we're working on it. we're already sending some custom dimensions ourselves on our site to test it. we need to work on how to present them in the dashboard next. it's a priority for us in terms of new feature development and will release it as soon as it's ready

**Reply by mattcameron** [reply]
Thanks @metmarkosaric. We're really only interested in querying it via the API. Any chance you want some beta testers?

**Reply by sptv2reg** [reply]
Frantically watching this issue. Saw a related commit the other day - are we getting close :-) ?

**Reply by florentdestremau** [reply]
Very interested too here. My use case: I have a multi-tenant SaaS so I want to track visits grouped by company (customer company). I would want to have 2 data visualisation possible (by querying or in the dashboard):

1. A filter per attribute on the main dashboard. Here I would choose a company
2. A cumulated graph by attribute on the metrics graph. A stacked graph of page visits per company for me, this would help me notice very active companies during a peak usage.

**Reply by ThomasTr** [reply]
Is this feature included in the v2.0.0-rc.0 release?

**Reply by NinjutsuSunrise** [reply]
My company would love to see this as well. Ideally, with GTM and dataLayer methods / docs available.

Thanks for all your hard work!

**Reply by danielfigueiredoff** [reply]
> Is this feature included in the v2.0.0-rc.0 release?

No info on changelog... Maybe under a feature flag?

**Comment by metmarkosaric** [comment]
We've now opened up a beta version of this to everyone. You can attach custom properties (also known as custom dimensions in Google Analytics) when sending a pageview to Plausible in order to create custom metrics. Instructions at https://plausible.io/docs/custom-pageview-props

**Reply by florentdestremau** [reply]
Great news, we're going to try out right away

**Reply by joseika** [reply]
Awesome!
Getting 404 on the script url provided in the docs though, @metmarkosaric:
https://plausible.io/script.pageview-props.js

**Reply by metmarkosaric** [reply]
@joseika this one? https://plausible.io/js/script.pageview-props.js. works all fine for me

**Reply by joseika** [reply]
@metmarkosaric found the error, «/js/» is missing in the example in this section: https://plausible.io/docs/custom-pageview-props#2-add-your-custom-properties-to-your-plausible-snippet

**Reply by metmarkosaric** [reply]
ahh yes! fixed now! thanks @joseika!

**Reply by florentdestremau** [reply]
Is there a way planned to have stacked graphs, charts or else of a page views per attribute? Like having the total views of the day stacked across a custom property?

**Reply by metmarkosaric** [reply]
@florentdestremau yes! before we release this publicly, we plan to display a list of all custom properties in the main view of the dashboard similar to how we list all the goals and custom events now. then when you filter the dashboard by any specific page, you'll be able to see all the custom properties sent alongside that page in that list

**Reply by florentdestremau** [reply]
Good to know. For now it's working great on our side for the last 2-3 hours. Very happy to see this come out :smile:

**Reply by dsager** [reply]
This is great, thanks for the update!

I have two questions regarding the property filter:
- Will it eventually be possible to filter for more than one property?
- Will it eventually be possible to use a `contains` filter in addition to `is` and `is not`?

Asking now, since it influences how we'd structure our page view properties...

**Reply by metmarkosaric** [reply]
@dsager both of these are on our to-do list but i cannot make any promise on when we'll get to work on them. the first one is more long-term as we'll need to wait for some changes to the database we use while the second one will probably be done faster. our current priority for custom props for pageviews is to list them all on the main view of the dashboard and then it's out of beta. the two ideas you shared are something we plan to do at a later stage

**Reply by dsager** [reply]
Thanks for the quick reply and info!

In case it helps, the use case for a `contains` filter is the following: Our news articles can have multiple categories. The initial implementation is to just store all categories in a single string, for example `{"news_topics": "Health,Education"}`. This means with a `property contains "Education"` filter I'd be able to get all article views on the topic of education.
I guess the alternative would be to have different fields `news_topic_1`, `news_topic_2` etc. But then this doesn't help much without being able to use multiple filters :)

Unless you have a better suggestion, I will just go with the current version (all categories in one string) and wait for the feature to drop :crossed_fingers:

**Reply by RobertJoonas** [reply]
Hey @dsager! Indeed, I agree that it's the best choice you have right now. Then once we have the contains filter ready, you'll be able to do something similar to what you can do with pages right now - select the contains filter type and pick multiple values to express a filter like `property 'news_topics' contains 'Health' OR 'Education'`.

**Reply by joseika** [reply]
Hi again! We've tested custom properties and and it works well in the dashboard. Is the same functionality available in the API?
We need to filter on a custom property and get all pageview metrics.

Best regards

**Reply by RobertJoonas** [reply]
Hi @joseika!

The functionality is mostly there in the API. The only limitation is that you cannot get any session level metrics at this stage. Those are `bounce_rate`, `visit_duration`, `visits`, and `views_per_visit`.

But here's what you **can do**:

* query for `visitors`, `pageviews` and `events` metrics
* use `is` and `is_not` filter types (by using `==` or `!=`)
* use `(none)` values in the query, e.g. `filters=event:props:author!=(none)` will return all events/pageviews that had the property specified
* break down by a custom prop in the breakdown endpoint (use `property=event:props:<your_property>`) as you would break down by any other property such as event:page.

**Reply by metmarkosaric** [reply]
**An update to the custom props:** There's now a dedicated 'Properties' tab in the 'Goal Conversions' report on the dashboard. There you can get a list view of all the different properties you're sending both on the custom event and on the pageview level. It makes it much easier to understand what's happening on your site.

In order for the properties to show up on your dashboard, you need to explicitly add them first in the 'Properties' section of the site settings (similar to how it works for custom events themselves). In the site settings you can add all the existing properties with one click. This also allows you the flexibility to decide what properties not to display on the dashboard (again similar to how it works for custom events themselves)

**Reply by SemaphoreOxalis** [reply]
Hi we have a very big issue there : with this addition we can't access anymore to our already collected custom props ? We really need theses because our clients have paid us for these ! I really hope it's not lost.
We currently have many studies based on these stats.
Another big issue with this approach : we collected information from search tools or other which create automatically the "custom" information we can't set manually the props.

Thanks for help on this because this put us in a very deep trouble :-/

**Reply by brandonburkett** [reply]
Hey, I tried to click `Already sending custom properties? Click to add all existing properties` on the domain settings page and now it is in a reloading loop (like window.location reloading over and over).  I had to close the tab.

**Reply by metmarkosaric** [reply]
hi @SemaphoreOxalis! you can add all the existing properties with a single click in the 'Properties' section of the site settings. then they will all show up in the new 'Properties' tab. does that work?

**Reply by SemaphoreOxalis** [reply]
Ok it seems to work now but : 

- The UI is very very confusing because when we clic in goal's tab on a goal it does not display under custom props and in Properties all customs are mixed together. Before when we click on a goal we have the custom prop attached to this goal in a tree view, it was very clear to read. Our client love this and paid us for that. And ourbound and file download still work like this
- This custom approach was included till now and I read it's going to be a premium feature ?? It seems a bit shady to move functions to another payment level :-/

As a long time user and evangelist I'm very confused and disappointed with this addition in the middle of the summer :-/

**Reply by metmarkosaric** [reply]
hey @brandonburkett! can you please clear cookies and site data for the plausible.io domain and then log back in into your account? this will solve the issue. thank you!

**Reply by brandonburkett** [reply]
@metmarkosaric Thx! Clearing cookie, local storage and session storage did it. I was able to add back our custom property without issue.

**Reply by metmarkosaric** [reply]
@SemaphoreOxalis 1. we've introduced custom properties on the pageview level so now there's a dedicated section for all properties. if you're not sending custom properties on the pageview level, then the list should be equal to what it was before but it will be displayed in the "Properties" tab rather than in the Goals tab. that's where's there's more space to show more data, have a 'details' view, choose not to show some properties etc. and you can still see the goals in the goals tab

2. custom properties for custom events will be free for existing subscribers and you'll be grandfathered to that when the business plans are live. now with the new dedicated section for properties we had to make the disclaimer but when the business plans are launched, it will not apply to any existing subscribers and you'll be able to continue using them without any change to the plan/price

**Reply by metmarkosaric** [reply]
@brandonburkett perfect, thank you for the update!

**Reply by sptv2reg** [reply]
Is it necessary (now or in the future) to actually add them to the dashboard in order for them to be registered? Or is this only a prerequisite for having them displayed on the dashboard?

**Reply by metmarkosaric** [reply]
hi @sptv2reg! it works the same way as custom events work. it is only a prerequisite for having them displayed on the dashboard. if you don't need the props displayed in the dashboard, you can simply start sending them our way without adding them in site settings

**Reply by SemaphoreOxalis** [reply]
@metmarkosaric Thanks for the clarification.
Since yesterday we have some weird issue on one of our client mixing page goal and custom goal+properties) . How could I send you the information about it ?

**Reply by metmarkosaric** [reply]
@SemaphoreOxalis you can contact our customer support with details https://plausible.io/contact

**Comment by mattcameron** [comment]
Hey team, great job in getting this feature out! 🙏 

I'm testing this out via the API and having trouble applying multiple filters for different custom properties - (adding an additional (AND) filter returns _more_  page views than with the single filter)

Is this supported yet? I can see in the dashboard that you can only select one at a time.

**Reply by metmarkosaric** [reply]
thanks @mattcameron! yeah, this is unfortunately not supported at this stage. it's something we plan to work on in the future but i cannot make any promise around when it may be ready. we'll update this thread as soon as we have some updates on our progress https://github.com/plausible/analytics/discussions/846

**Reply by mattcameron** [reply]
Okay, thanks @metmarkosaric!

**Comment by k9ordon** [comment]
Fantastic addition! I was curious to know if there are any plans to integrate this feature with the [plausible-tracker](https://github.com/plausible/plausible-tracker)?

**Comment by Jelledb** [comment]
I would really love to use this. Is this available for the self-hosted variant of Plausible?

My use cases are: 

1) Track which customer name the pageview came from (multi-tenant SaaS use case)
2) Track whether the respective user is an admin or a regular user., just as a simple true/false.

**Reply by timcole** [reply]
Yes, this was made available in self-hosted v2.0.0

**Comment by SemaphoreOxalis** [comment]
Hi,
After many clients review we want to share a common feedback : we need the goal attachment name after the properties name. For example we have a "Search" goal with a "Terms" properties. Many clients appreciate to have something like "Terms (Search)" in the properties dropzone.
The new Goals/Properties/Funnel views are quite confusing (they really dislike this new one) and they understand they have to select goals for filtering properties but the addition of the Goal after the properties terms permit them to review all properties corectly and understand the use of each of them.

Thanks for this.
